README.md

```diff
@@ -1,24 +1,70 @@
-# README
+# Recipe App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-Things you may want to cover:
+## Getting Started
 
-* Ruby version
+To get a local copy of the project, follow these steps: 
+1. Open the terminal and navigate to the folder where you would like to put these files.
+2. Type the following commands into the terminal: 
+ ```
+ git clone https://github.com/jsug9/recipe-app
+ ```
+ ```
+ cd recipe-app
+ ```
 
-* System dependencies
+Check the ruby version being used and install it if it is not the same with that of the local machine
+Set up database direct them to the config/database.yml file
+```
+bundle install
+```
+```
+'npm install' or 'yarn install'
+```
+```
+rails db:create
+```
+```
+rails db:migrate
+```
+```
+rails 'db:seed' if there is data in the seed
+```
+```
+'rails s' to start the server locally
+```
 
-* Configuration
+To execute the tests, run the following command in your terminal:
+```
+rspec
+```
 
-* Database creation
+## Author
 
-* Database initialization
+👤 **Augusto Galindo**
 
-* How to run the test suite
+- GitHub: [@jsug9](https://github.com/jsug9)
+- LinkedIn: [Augusto Galindo](https://www.linkedin.com/in/augustogalindo/)
 
-* Services (job queues, cache servers, search engines, etc.)
+👤 **Aaron Keegan**
 
-* Deployment instructions
+- GitHub: [@AKeeganDev](https://github.com/AKeeganDev)
 
-* ...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jsug9/recipe-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to my peers who are always there to offer support.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
**Optional**
- Please fix the MIT link for a professional README file.

Reply:
Fixed ✅